go.mod

```diff
@@ -1,70 +1,68 @@
 module github.com/kserve/modelmesh-serving
 
-go 1.18
+go 1.19
 
 require (
 	github.com/dereklstinson/cifar v0.0.0-20200421171932-5722a3b6a0c7
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/protobuf v1.5.2
-	github.com/google/go-cmp v0.5.8
-	github.com/kserve/kserve v0.9.1-0.20221013012311-335dfbcc461a
+	github.com/google/go-cmp v0.5.9
+	github.com/kserve/kserve v0.9.0
```

Comment:
can you change the `kserve` version to [`v0.10.0-rc0`](https://github.com/kserve/kserve/releases/tag/v0.10.0-rc0)

```shell
$ go get github.com/kserve/kserve@v0.10.0-rc0

go: downloading github.com/kserve/kserve v0.10.0-rc0
go: upgraded github.com/kserve/kserve v0.9.1-0.20221013012311-335dfbcc461a => v0.10.0-rc0

```